docker/hoodie/hadoop/historyserver/NOTICE

```diff
@@ -0,0 +1,5 @@
+Apache HUDI
```

Comment:
nit: "Apache Hudi" everywhere? (just in case this becomes an issue) @bvaradar

Reply:
Thanks @vinothchandar. Made this change.